test/runtests.jl

```diff
@@ -1,44 +1,12 @@
-using EDF
-using EDF: TimestampedAnnotationList, PatientID, RecordingID, SignalHeader,
-           Signal, AnnotationsSignal
-using Dates
-using FilePathsBase
-using Test
+include("set_up_tests.jl")
```

Comment:
Why make this change? Doesn't feel like it buys a whole lot, just kind of makes it mildly inconvenient to determine what the test setup is.

Reply:
this matches a pattern widespread through Beacon and makes interactive testing a lot easier. Instead of scrolling to the top of this file, I can just focus on the tests in question and do

```julia
using TestEnv
TestEnv.activate()
include("test/set_up_tests.jl")
```